### ISSUE TITLE
String treated as Timestamp in some special cases

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#378]
```
While adopting our own persistance layer to use H2, I discoverd a strange 
behaviour. Attached is a test case which is extracted from our test suite. 

It contains some tables, some data and two views. Selecting data from one of 
this views results in a SQLException. H2 seems to try to convert a string value 
to timestamp. The string value is a string in the DB, no need to convert it to 
a timestamp.


```

Original issue reported on code.google.com by `andreas....@gmail.com` on 16 Feb 2012 at 4:57

**Comment by GoogleCodeExporter** [comment]
```
I will have a look.
```

Original comment by `thomas.t...@gmail.com` on 17 Feb 2012 at 8:31
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This will be fixed in the next release. Thanks for your help!
```

Original comment by `thomas.t...@gmail.com` on 30 Apr 2012 at 4:53

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.3.167
```

Original comment by `thomas.t...@gmail.com` on 23 May 2012 at 7:23
- Changed state: **Fixed**
